libafl/src/corpus/ondisk.rs

```diff
@@ -229,7 +229,9 @@ where
             };
 
             let filename_str = filename.to_str().expect("Invalid Path");
-            testcase.set_filename(filename_str.into());
+            testcase
+                .set_filename(filename_str.into())
+                .expect("Could not save testcase to disk");
```

Comment:
Don't use `expect` here, we want to return an Err result (using `?`)

Reply:
fixed